DocService/sources/DocsCoServer.js

```diff
@@ -3289,7 +3289,9 @@ exports.commandFromServer = function (req, res) {
             }
             break;
           case 'forcesave':
-            let forceSaveRes = yield startForceSave(docId, commonDefines.c_oAscForceSaveTypes.Command, params.userdata, undefined, undefined, undefined, undefined, utils.getBaseUrlByRequest(req));
+            // If async was not send - set true (as default)
+            let async = (typeof params.async === 'undefined') ? true : (typeof params.async === 'string') ? 'true' === params.async : params.async;

```

Comment:
by analogy with https://api.onlyoffice.com/editors/conversionapi#async, `async ` should be boolean only

Reply:
This ternary based on this code for async variable:
https://github.com/ONLYOFFICE/server/blob/master/DocService/sources/converterservice.js#L266
https://github.com/ONLYOFFICE/server/blob/master/DocService/sources/converterservice.js#L328

Reply:
unfortunately this PR is not enough. parameter does not work. sync cycle monitors another key

Reply:
Hmm. I see variable `docId` which used in cycle `selectRes = yield taskResult.select(docId);` (136 line) is correct. Its same key which sended in command.

Reply:
Any news?